### PR TITLE
Add support Yandex Alice

### PR DESCRIPTION
**Pull request by Klavishnik**
Please sign (check) the below before submitting the Pull Request:

-  I have signed the ntop Contributor License Agreement at https://github.com/ntop/legal/blob/main/individual-contributor-licence-agreement.md
-  I have read the contributing guidelines at https://github.com/ntop/nDPI/blob/dev/CONTRIBUTING.md
-  I have updated the documentation (in doc/) to reflect the changes made (if applicable)

Link to the related [issue](https://github.com/ntop/nDPI/issues):

Describe changes:
Add support Yandex Alice

### CODE REVIEW

File: src/lib/ndpi_content_match.c.inc [hunk 1]
```diff
@@ -1410,6 +1410,12 @@ static ndpi_protocol_match host_match[] =
...
    { "direct.yandex.",                   "YandexDirect", NDPI_PROTOCOL_YANDEX_DIRECT, CUSTOM_CATEGORY_ADVERTISEMENT, NDPI_PROTOCOL_TRACKER_ADS, NDPI_PROTOCOL_DEFAULT_LEVEL },
+   
+   { "quasar.yandex.ru",                 "YandexAlice", NDPI_PROTOCOL_YANDEX_ALICE, NDPI_PROTOCOL_CATEGORY_VIRTUAL_ASSISTANT, NDPI_PROTOCOL_ACCEPTABLE, NDPI_PROTOCOL_DEFAULT_LEVEL },
+   { "scbh.yandex.net",                  "YandexAlice", NDPI_PROTOCOL_YANDEX_ALICE, NDPI_PROTOCOL_CATEGORY_VIRTUAL_ASSISTANT, NDPI_PROTOCOL_ACCEPTABLE, NDPI_PROTOCOL_DEFAULT_LEVEL },
+   { "report.appmetrica.yandex.net",     "YandexAlice", NDPI_PROTOCOL_YANDEX_ALICE, NDPI_PROTOCOL_CATEGORY_VIRTUAL_ASSISTANT, NDPI_PROTOCOL_ACCEPTABLE, NDPI_PROTOCOL_DEFAULT_LEVEL },
+   { "uniproxy.alice.yandex.net",        "YandexAlice", NDPI_PROTOCOL_YANDEX_ALICE, NDPI_PROTOCOL_CATEGORY_VIRTUAL_ASSISTANT, NDPI_PROTOCOL_ACCEPTABLE, NDPI_PROTOCOL_DEFAULT_LEVEL },
+   { "quasar.yandex.ru",                 "YandexAlice", NDPI_PROTOCOL_YANDEX_ALICE, NDPI_PROTOCOL_CATEGORY_VIRTUAL_ASSISTANT, NDPI_PROTOCOL_ACCEPTABLE, NDPI_PROTOCOL_DEFAULT_LEVEL },
```
Comment (IvanNardi) [inline src/lib/ndpi_content_match.c.inc]: Duplicate. Remove it

**Review by IvanNardi — CHANGES_REQUESTED** [review]
Thanks for your contribution. Could you please:
* sign the CLA?
* add an example of this traffic? One simple TLS flow (the first 6-7 pkts only) to any of this domains is enough. It would be wonderful if you could merge it with the existing `tests/pcaps/sites2.pcap` file

File: src/lib/ndpi_content_match.c.inc (hunk 1 above)
Comment (Klavishnik) [reply src/lib/ndpi_content_match.c.inc]: Hello!
Yandex Alice is a voice assistant primarily used in smart stations, although its features are also available on the website, browser, and Yandex app. Perhaps it’s not necessary to merge the test example of Yandex Alice traffic with the `sites2.pcap` file?

I’m attaching a sample of captured traffic as a file, along with the result of its analysis using `ndpiReader`.

[yandex-alice.zip](https://github.com/user-attachments/files/17952236/yandex-alice.zip)

Comment (Klavishnik) [reply src/lib/ndpi_content_match.c.inc]: Remove duplicate in new commit

Comment (Klavishnik) [reply src/lib/ndpi_content_match.c.inc]: I signed the CLI through the google form

**Comment by IvanNardi** [comment]
Thanks for your contribution, @Klavishnik

**Pull request merged**